etc/alembic_offline.ini.template

```diff
@@ -0,0 +1,78 @@
+# Copyright 2014-2019 CERN for the benefit of the ATLAS collaboration.
```

Comment:
Cosmetic, but please change this header to the current one. :smile

Reply:
Should I also remove the authors as in the new header style?

Reply:
Yes 😄